mixer/docker/Dockerfile.mixer

```diff
@@ -1,5 +1,21 @@
+# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
+ARG BASE_DISTRIBUTION=default
+
+# Version is the base image version from the TLD Makefile
+ARG BASE_VERSION=latest
+
+# The following section is used as base image if BASE_DISTRIBUTION=default
+FROM docker.io/istio/base:${BASE_VERSION} as default
+
+USER 1337:1337
```

Comment:
How does this work with UDS socket between proxy and mixer in mixer's pod? Did you test that file permissions do not interfere with UDS?

Reply:
Thanks for your comment. Just noticed that the mixer is also using UDS socket. I think I need to apply the same fix (https://github.com/istio/istio/pull/15947) in mixer side. Could you please provide your comment for that PR as well. BTW, do you know where are still using UDS socket?

Reply:
@kyessenov I just noticed that `uds-socket` is created from emptyDir and the permission is always 777. so it can support run as non root.

Reply:
@kyessenov I just noticed that uds-socket is created from emptyDir and the permission is always 777. so it can support run as non root.